.github/workflows/ci.yml

```diff
@@ -1,135 +0,0 @@
-    branches:
-      - main
-jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 16
-      - uses: actions/setup-go@v2
-        with:
-          go-version: 1.18
-      - name: Install dependencies
-        run: |
-          npm install --global yarn
-          make setup
-          make init-dep
-      - name: Lint source code
-        run: make lint
-
-  integration-test:
-    runs-on: ubuntu-latest
-    needs: lint
-    services:
-      postgres:
-        image: postgres:12.4
-        env:
-          POSTGRES_DB: mlp
-          POSTGRES_USER: mlp
-          POSTGRES_PASSWORD: mlp
-        ports:
-          - 5432:5432
-      keto:
-        image: oryd/keto:v0.4.3-alpha.2
-        env:
-          DSN: memory
-        ports:
-          - 4466:4466
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
-        with:
-          go-version: 1.18
-      - name: Integration test
-        env:
-          DATABASE_HOST: localhost
-          DATABASE_NAME: mlp
-          DATABASE_USER: mlp
-          DATABASE_PASSWORD: mlp
-          KETO_URL: http://localhost:4466
-        run: make it-test-api-ci
-
-  publish-gojek-mlp-ui:
-    if: ${{ startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-latest
-    needs: lint
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 16
-          registry-url: https://registry.npmjs.org
-      - name: Install dependencies
-        working-directory: ./ui
-        run: yarn install
-      - name: Publish @gojek/mlp-ui library
-        working-directory: ./ui
-        run: |
-          yarn set-version-from-git
-          yarn lib publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-
-  publish-mlp-docker:
-    if: ${{ startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-latest
-    needs: integration-test
-    steps:
-      - uses: actions/checkout@v2
-      - name: Build and push docker image
-        uses: docker/build-push-action@v1
-        with:
-          username: ${{ github.actor }}
-          password: ${{ secrets.GHCR_TOKEN }}
-          registry: ghcr.io
-          tag_with_ref: true
-
-  e2e-test:
-    if: ${{ startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-latest
-    needs: publish-mlp-docker
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
-        with:
-          go-version: 1.18
-      - name: Set up Helm
-        uses: azure/setup-helm@v1
-        with:
-          version: v3.8.1
-      - name: Create kind cluster
-        uses: helm/kind-action@v1.3.0
-      - name: Create mlp namespace
-        run: kubectl create namespace mlp
-      - name: Deploy MLP
-        run: ./scripts/e2e/deploy-mlp.sh "charts/mlp"
-      - name: Run E2E test
-        run: ./scripts/e2e/run-e2e.sh "api"
-
-  release-charts:
```

Comment:
Is my understanding correct that we no longer need to release the chart from this repo because it's covered in the common charts repo? https://github.com/caraml-dev/helm-charts/tree/main/charts

Reply:
Yes, I'll remove the chart from this repo completely after our internal deployment pipeline is updated to support this transition.